pywbem/_server.py

```diff
@@ -762,6 +764,34 @@ def get_central_instances(self, profile_path, central_class=None,
             implementation supports only the scoping class methodology.
             `None` will cause the scoping class methodology not to be attempted.
 
+          reference_direction (:term:`string`):
+            Defines the usage of the ResultRole parameter in the association
+            request that determines the relation between component registered
+            profiles.  The possible values are 'dmtf' and 'snia' the default is
+            'dmtf'. This parameter exists because of issues between DMTF and
+            SNIA in the the definition of CIM_ReferencedProfile and its
+            subclasses.
```

Comment:
I would not talk about "issues", but about different definitions without judging it.

Reply:
good catch. Fixed